<launch>

<include file="$(find-pkg-share very_simple_robot_simulator)/launch/minimal_simulator_py.xml" >
    <arg name="initial_x" value="0.5" />
    <arg name="initial_y" value="0.5" />
    <arg name="initial_yaw" value="0.0" />
  </include>

  <node pkg="very_simple_robot_simulator" name="world_state_gui" exec="world_state_gui.py" output="screen" >
    <param name="map_file" value="$(find-pkg-share particle_filter)/mapa.yaml" />
  </node>

  <include file="$(find-pkg-share very_simple_robot_simulator)/launch/lidar_simulator.xml" />

    <node pkg="lab_3" exec="robot_reactivo_test_C" name="robot_reactivo_test_C" >
        <param name="file_path" value="robotica_movil/src/lab_3/lab_3/mapa.pgm" />
    </node>
    <node pkg="lab_3" exec="localizacion_test_C" name="localizacion_test_C" >
            <param name="ruta" value="robotica_movil/src/lab_3/lab_3/" />
    </node>
</launch>